core/src/main/java/com/google/errorprone/bugpatterns/javadoc/EmptyBlockTag.java

```diff
@@ -0,0 +1,129 @@
+import com.google.errorprone.bugpatterns.BugChecker.ClassTreeMatcher;
+import com.google.errorprone.bugpatterns.BugChecker.MethodTreeMatcher;
+import com.google.errorprone.bugpatterns.BugChecker.VariableTreeMatcher;
+import com.google.errorprone.matchers.Description;
+import com.sun.source.doctree.BlockTagTree;
+import com.sun.source.doctree.DeprecatedTree;
+import com.sun.source.doctree.DocTree;
+import com.sun.source.doctree.ParamTree;
+import com.sun.source.doctree.ReturnTree;
+import com.sun.source.doctree.ThrowsTree;
+import com.sun.source.tree.ClassTree;
+import com.sun.source.tree.MethodTree;
+import com.sun.source.tree.VariableTree;
+import com.sun.source.util.DocTreePath;
+import com.sun.source.util.DocTreePathScanner;
+import java.util.List;
+
+/**
+ * Matches block tags ({@literal @}param, {@literal @}return, {@literal @}throws,
+ * {@literal @}deprecated) with an empty description.
+ *
+ * @author andrewash@google.com (Andrew Ash)
+ */
+@BugPattern(
+    name = "EmptyBlockTag",
+    summary = "A block tag (@param, @return, @throws, @deprecated) has an empty description.",
+    severity = WARNING,
+    linkType = CUSTOM,
+    link = "http://google.github.io/styleguide/javaguide.html#s7.1.3-javadoc-block-tags",
+    tags = StandardTags.STYLE,
+    providesFix = ProvidesFix.REQUIRES_HUMAN_ATTENTION,
+    documentSuppression = false)
+public final class EmptyBlockTag extends BugChecker
+    implements ClassTreeMatcher, MethodTreeMatcher, VariableTreeMatcher {
+
+  @Override
+  public Description matchClass(ClassTree classTree, VisitorState state) {
+    checkForEmptyBlockTags(state);
+    return Description.NO_MATCH;
+  }
+
+  @Override
+  public Description matchMethod(MethodTree methodTree, VisitorState state) {
+    checkForEmptyBlockTags(state);
+    return Description.NO_MATCH;
+  }
+
+  @Override
+  public Description matchVariable(VariableTree variableTree, VisitorState state) {
+    checkForEmptyBlockTags(state);
+    return Description.NO_MATCH;
+  }
+
+  private void checkForEmptyBlockTags(VisitorState state) {
+    DocTreePath path = Utils.getDocTreePath(state);
+    if (path != null) {
+      new EmptyBlockTagChecker(state).scan(path, null);
+    }
+  }
+
+  private final class EmptyBlockTagChecker extends DocTreePathScanner<Void, Void> {
+    private final VisitorState state;
+
+    private EmptyBlockTagChecker(VisitorState state) {
+      this.state = state;
+    }
+
+    @Override
+    public Void visitParam(ParamTree paramTree, Void unused) {
+      reportMatchIfEmpty(paramTree, paramTree.getDescription());
+      return super.visitParam(paramTree, null);
+    }
+
+    @Override
+    public Void visitReturn(ReturnTree returnTree, Void unused) {
+      reportMatchIfEmpty(returnTree, returnTree.getDescription());
+      return super.visitReturn(returnTree, null);
+    }
+
+    @Override
+    public Void visitThrows(ThrowsTree throwsTree, Void unused) {
+      reportMatchIfEmpty(throwsTree, throwsTree.getDescription());
```

Comment:
There may be one case in which deletion is not the best suggestion: in case of a `@throws` tag without a description, where the declared exception is unchecked and not part of the documented method's signature. In that case it may be better to only flag the issue, forcing the developer to either manually deleted it (if not applicable) or add documentation (otherwise).